package.json

```
@@ -3,7 +3,7 @@
   "version": "0.0.1",
   "private": true,
   "scripts": {
-    "clean": "rm -rf node_modules && yarn",
+    "clean": "yarn cache clean && watchman watch-del-all && rm -rf node_modules && yarn && rm ios/Podfile.lock && yarn install:pod && react-native start --resetCache",
```

```suggestion
    "clean": "yarn cache clean && watchman watch-del-all && rm -rf node_modules && rm ios/Podfile.lock && yarn && yarn install:pod",
    "start": "react-native start --resetCache"
    "clean-and-start": "yarn clean && yarn start"
```
minor suggestion, but I find having `clean` initiate metro bundler to be counter intuitive